plugins/pdr_deterministic_plugin/ufm_sim_web_service/telemetry_collector.py

```diff
@@ -0,0 +1,27 @@
+import logging
```

Comment:
Can you please remove this file for now?
The collection telemetry task will change it content and will not be merged yet.